cea/globalvar.py

```diff
@@ -39,11 +39,11 @@ def __init__(self):
         self.Es = 0.9  # franction of GFA that has electricity in every building
         # constant values for HVAC
         self.nrec_N = 0.75  # possible recovery
-        self.NACH_inf_non_occ = 0.2  # num air exchanges due to infiltration when no occupied
-        self.NACH_inf_occ = 0.5  # num air exchanges due to infiltration when occupied
+        #self.NACH_inf_non_occ = 0.2  # num air exchanges due to infiltration when no occupied
```

Comment:
why not just delete it?

Reply:
because I'm not sure whether all the code is tested. Maybe these things are used somewhere I don't know!
My policy would be to comment out everything until finally the whole `gv` can be deleted.
I saw the neural network stuff uses quite a bit of functions from demand.